src/yarn/monorepo-release.ts

```diff
@@ -72,6 +73,17 @@ export class MonorepoRelease extends Component {
         release: {
           workspace: workspaceRelease.workspace,
           publisher: workspaceRelease.publisher,
+          options: {
+            // enforced options
+            workflowName: this.options.releaseWorkflowName,
+            tagPrefix: `${project.name}@`,
+
+            // options that may be override locally
+            majorVersion: options.versionBranchOptions.majorVersion ?? this.options.majorVersion,
+            minMajorVersion: options.versionBranchOptions.minMajorVersion ?? this.options.minMajorVersion,
+            prerelease: options.versionBranchOptions.prerelease ?? this.options.prerelease,
+            npmDistTag: options.npmDistTag ?? this.options.npmDistTag,
```

Comment:
We prefer the project local option here over the monorepo option. But only `prerelease` and `npmDistTag` actually have an effect.